homeassistant/components/overkiz/sensor.py

```diff
@@ -153,108 +158,93 @@ class OverkizSensorDescription(SensorEntityDescription):
         key=OverkizState.CORE_LUMINANCE,
         name="Luminance",
         device_class=SensorDeviceClass.ILLUMINANCE,
-        # core:MeasuredValueType = core:LuminanceInLux
-        native_unit_of_measurement=LIGHT_LUX,
+        native_unit_of_measurement=LIGHT_LUX,  # core:MeasuredValueType = core:LuminanceInLux
```

Comment:
Can you please adjust the comments so they are back to how they were before?
Since this is such an old PR, it's likely that they have been moved to fix line-length issues.

Reply:
Sure. I have applied the modifications.